### ISSUE TITLE
[bayesian-priors] No tests (Trac #1148)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1142]
This project has no tests. 0% code coverage is pretty bad.

Migrated from https://code.icecube.wisc.edu/ticket/1148
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:57", 
    "description": "This project has no tests. 0% code coverage is pretty bad.", 
    "reporter": "olivas", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067117911749", 
    "component": "combo reconstruction", 
    "summary": "[bayesian-priors] No tests", 
    "priority": "critical", 
    "keywords": "", 
    "time": "2015-08-18T00:41:05", 
    "milestone": "", 
    "owner": "kjmeagher", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150831T16:32:16: 
* olivas changed status from "new" to "assigned"
* olivas changed owner from "claudio.kopper" to "olivas"

**Comment by icecube-trac** [comment]
Trac update at 20150908T19:48:18: olivas changed status from "assigned" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20151014T12:54:37: kjmeagher changed owner from "olivas" to "kjmeagher"

**Comment by icecube-trac** [comment]
Trac update at 20151130T13:40:27: 
* kjmeagher commented:

> In changeset 139947:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="139947"
> move test to tests directory, delete unused 42.dat. code coverage now ~80%, fixes #1142
> ```


* kjmeagher changed resolution from "" to "fixed"
* kjmeagher changed status from "accepted" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:57: 
* olivas commented:

> Milestone IceRec V5 deleted


* olivas changed milestone from "IceRec V5" to ""
